CHANGELOG.md

```diff
@@ -2,6 +2,26 @@
 
 ## v37.0.0-SNAPSHOT - unreleased
 
+### 💥 Breaking Changes
+* New `GridModel` config `colChooserModel` replaces `enableColChooser` to allow for more flexible
```

Comment:
love this documentation -- a little more detailed than usual for changelog, but really clear. Should we move it to a jsdoc  for colChooserModel